resolwe_bio/processes/support_processors/seqtk_reverse_complement.py

```diff
@@ -0,0 +1,134 @@
+"""Reverse complement reads with Seqtk."""
+import os
+
+from resolwe.process import Process, DataField, FileField, Cmd, ListField, StringField
+
+
+class ReverseComplementSingle(Process):
+    """Reverse complement FASTQ reads file using Seqtk."""
+
+    slug = 'seqtk-rev-complement-single'
+    process_type = 'data:reads:fastq:single:seqtk:comp'
+    name = 'Reverse complement FASTQ'
+    requirements = {
+        'expression-engine': 'jinja',
+        'executor': {
+            'docker': {
+                'image' : 'resolwebio/common:1.1.0'
+            },
+        },
+        'resources': {
+            'cores': 1,
+            'memory': 16384,
```

Comment:
What is the memory footprint of this process? We will be dealing with really large files ~10GB and more, so make sure that reserved memory is sufficient. It is difficult to perform such test locally, but did you maybe checked how memory requirements grow when increasing file size? Linearly, constant?

Reply:
I was only able to test it locally on read files with 5GB size.  According to the results of [this](https://bioinf.shenwei.me/seqkit/benchmark/) benchmarking the RAM usage is usually smaller than the file size.

Reply:
OK.